.github/workflows/pr-closed.yml

```diff
@@ -0,0 +1,32 @@
+name: 🎉 PR closed
```

Comment:
```suggestion
name: PR closed
```

Reply:
No, let there be emojis in the Workflows names, I would rather change the emoji here

Reply:
Here's a clear example of why emojis are useful:

![image](https://github.com/user-attachments/assets/3d731039-cb4a-489f-a602-af444c633560)

Thanks to them, it is much easier and more intuitive to understand what it does

Reply:
Hmm, those probably would help contributors... Sure, I'll give them a go.